File: internal/content/testdata/bench/javascript/nc-two-sum-ii.js
function two_sum_ii(numbers, target) {
    let i = 0, j = numbers.length - 1;
    while (i < j) {
        const total = numbers[i] + numbers[j];
        if (total === target) return [i + 1, j + 1];
        if (total < target) i++;
        else j--;
    }
    return [];
}

File: internal/content/testdata/bench/go/nc-two-sum-ii.go
func two_sum_ii(numbers []int, target int) []int {
	i, j := 0, len(numbers)-1
	for i < j {
		total := numbers[i] + numbers[j]
		if total == target {
			return []int{i + 1, j + 1}
		} else if total < target {
			i++
		} else {
			j--
		}
	}
	return []int{}
}
